app/models/account_request.rb

```diff
@@ -0,0 +1,2 @@
+class AccountRequest < ActiveRecord::Base
```

Comment:
Shouldn't this model replace the RequestedUser model? I think the validations need to be moved over from that file

Reply:
We have made the required change in the latest commit as per your suggestion. Please take a look and let us if it is fine.